.github/workflows/checkstyle.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - name: Install dependencies
         shell: bash
-        run: dnf --setopt install_weak_deps=False install -y gettext git-core jss
+        run: dnf --setopt install_weak_deps=False install -y gettext git-core
```

Comment:
With the removal of JSS we no longer need fedora container in our CIs  and if i remebered correctly you can also delete instalation of docker, because it is installed on the ubuntu runner by default.

Reply:
So added